lumigator/backend/backend/services/jobs.py

```diff
@@ -229,7 +229,7 @@ def _add_dataset_to_db(self, job_id: UUID, request: JobCreate, s3: S3FileSystem)
 
         loguru.logger.info(f"Dataset '{dataset_filename}' with ID '{dataset_record.id}' added to the database.")
 
-    def _validate_results(self, job_id: UUID, s3: S3FileSystem) -> JobResultObject:
+    def _validate_results(self, job_id: UUID, request: JobCreate, s3: S3FileSystem):
```

Comment:
AFAICT the request is not used within this method?

Reply:
And this also introduces a bug cause it's called like this in 194:

```
results = self._validate_results(job_id, s3)
```